components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -1,9 +1,10 @@
+
 import { default as Icon } from "@spectrum-css/icon/stories/icon.stories.js";
 import { version } from "../package.json";
-import { Template } from "./template";
+import { CustomIconTemplate, SizingTemplate, Template } from "./template";
 
 /**
- * The picker button component is used as a dropdown trigger. See Combobox.
+ * The picker button component is used as a dropdown trigger within other components such as combobox.
```

Comment:
Nitpick: we could link to combobox here.

Reply:
I've added that link.